services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -386,7 +441,27 @@ def compute_descriptive_statistics_response(
     con.sql("INSTALL httpfs")
```

Comment:
do we have to free the memory at the end of the job by deleting the table that was created ?